deployment/CP01CNPM_Deploy.csv

```diff
@@ -26,3 +26,12 @@ AR39,,AR44,,CP01CNPM-WFP01-02-DOFSTK000,3,1,2019-10-07T13:44:00,2020-06-14T12:23
 AR39,,AR44,,CP01CNPM-WFP01-03-CTDPFK000,3,1,2019-10-07T13:44:00,2020-06-14T12:23:00,CGMCP-01CNPM-00003,CGVEH-CWFPCP-30002,CGINS-CTDPFK-00185,40.139158,-70.779413,,111,132,
 AR39,,AR44,,CP01CNPM-WFP01-04-FLORTK000,3,1,2019-10-07T13:44:00,2020-06-14T12:23:00,CGMCP-01CNPM-00003,CGVEH-CWFPCP-30002,CGINS-FLORTK-01538,40.139158,-70.779413,,111,132,
 AR39,,AR44,,CP01CNPM-WFP01-05-PARADK000,3,1,2019-10-07T13:44:00,2020-06-14T12:23:00,CGMCP-01CNPM-00003,CGVEH-CWFPCP-30002,CGINS-PARADK-20542,40.139158,-70.779413,,111,132,
+AR48,,,,CP01CNPM-RII01-02-ADCPTG010,4,1,2020-10-28T00:00:00,,CGMCP-01CNPM-00004,,CGINS-ADCPTG-24544,40,-70,,,,
+AR48,,,,CP01CNPM-SBS01-00-STCENG000,4,1,2020-10-28T00:00:00,,CGMCP-01CNPM-00004,,CGCON-BSTC01-00013,40,-70,,0,,
+AR48,,,,CP01CNPM-SBS01-01-MOPAK0000,4,1,2020-10-28T00:00:00,,CGMCP-01CNPM-00004,,CGINS-MOPAK0-11360,40,-70,,0,,
+AR48,,,,CP01CNPM-WFP01-00-WFPENG000,4,1,2020-10-28T00:00:00,,CGMCP-01CNPM-00004,CGVEH-CWFPCP-30002,CGCON-CWFPCP-30002,40,-70,,,,
+AR48,,,,CP01CNPM-WFP01-01-VEL3DK000,4,1,2020-10-28T00:00:00,,CGMCP-01CNPM-00004,CGVEH-CWFPCP-30002,CGINS-VEL3DK-00470,40,-70,,,,
+AR48,,,,CP01CNPM-WFP01-02-DOFSTK000,4,1,2020-10-28T00:00:00,,CGMCP-01CNPM-00004,CGVEH-CWFPCP-30002,CGINS-DOFSTK-03523,40,-70,,,,
+AR48,,,,CP01CNPM-WFP01-03-CTDPFK000,4,1,2020-10-28T00:00:00,,CGMCP-01CNPM-00004,CGVEH-CWFPCP-30002,CGINS-CTDPFK-00185,40,-70,,,,
+AR48,,,,CP01CNPM-WFP01-04-FLORTK000,4,1,2020-10-28T00:00:00,,CGMCP-01CNPM-00004,CGVEH-CWFPCP-30002,CGINS-FLORTK-01538,40,-70,,,,
+AR48,,,,CP01CNPM-WFP01-05-PARADK000,4,1,2020-10-28T00:00:00,,CGMCP-01CNPM-00004,CGVEH-CWFPCP-30002,CGINS-PARADK-20609,40,-70,,,,
```

Comment:
Looks good. Confirmed SNs with config sheet

Reply:
Looks good.  Compared to Config Sheet, WFP QCT 3310-00100-00127, and WFP instrument cal docs.